openehr-rm/src/main/java/com/nedap/archie/rminfo/UpdatedValueHandler.java

```diff
@@ -85,6 +87,10 @@
                 }
             }
         }
+        if(ordinal.getSymbol() != null && ordinal.getSymbol().getDefiningCode() != null) {
+            //also fix the DvCodedText inside the DvOrdinal
+            result.putAll(fixDvCodedText(rmObject, archetype, pathOfParent));
+        }
```

Comment:
The definingCode is always set here, otherwise this method wouldn't have been called.
```suggestion
        //also fix the DvCodedText inside the DvOrdinal
        result.putAll(fixDvCodedText(rmObject, archetype, pathOfParent));
```

Reply:
I could do it, but this could also cause future issues. Do you mind if I leave it here?